packages/cos/build.yaml

```diff
@@ -19,10 +19,17 @@ requires:
 - name: "grub-config"
   category: "system"
   version: ">=0"
+
+# Skip selinux policies for recovery
```

Comment:
recovery shouldn't need selinux as its not running any core services. At this point it's dropped and only available for the system/cos. We might want to do other changes there as well, regarding for example a split cloud-config set. But I think it's out of purpose of this PR, and we can do a follow up later to iterate on the recovery image with #189